base/toml_parser.jl

```diff
@@ -1,5 +1,9 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+`Base.TOML` is an internal, undocumented TOML parser;
+users should call the TOML.jl standard library instead.
```

Comment:
Technically, it's not a separate parser — it's actually used in TOML.jl — but it's
an internal part of the implementation.  Maybe:
```suggestion
`Base.TOML` is an undocumented internal part of Julia's TOML parser
implementation.  Users should call the the documented interface in the
TOML.jl standard library instead (by `import TOML` or `using TOML`).
```

Reply:
Yes, that will be more accurate. Thanks!